proto/hypervisor/messages.go

```diff
@@ -849,6 +849,7 @@ type VmInfo struct {
 	DisableVirtIO       bool        `json:",omitempty"`
 	ExtraKernelOptions  string      `json:",omitempty"`
 	Hostname            string      `json:",omitempty"`
+	HypervisorHostname  string      `json:",omitempty"`
```

Comment:
Also, this type should not be changed, since it's effectively owned by the Hypervisor. Instead, the internal `vmInfoType` type in the Fleet Manager should be changed.